src/sentry/conf/server.py

```diff
@@ -737,6 +737,8 @@ def SOCIAL_AUTH_DEFAULT_USERNAME() -> str:
     "sentry.data_export.tasks",
     "sentry.discover.tasks",
     "sentry.hybridcloud.tasks.deliver_webhooks",
+    "sentry.hybridcloud.tasks.backfill_outboxes",
+    "sentry.hybridcloud.tasks.deliver_from_outbox",
```

Comment:
The tasks didn't have their names changed, so updating `CELERY_IMPORTS` should be all that is necessary for tasks to continue executing without interruption.